### PR TITLE
Generate ext/intl class synopses from stubs - part 3

### PR DESCRIPTION
**Pull request by kocsismate**
Only formatting changes (+ fallbacks)

### CODE REVIEW

File: reference/intl/resourcebundle.xml [hunk 1]
```diff
@@ -43,20 +43,27 @@
...
-    </classsynopsisinfo>
-    <!-- }}} -->
 
-    <classsynopsisinfo role="comment">Methods</classsynopsisinfo>
-    <xi:include
-     xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.resourcebundle')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[1])" />
-    <xi:include
-     xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.resourcebundle')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
+     <oointerface>
+      <interfacename>IteratorAggregate</interfacename>
```
Comment (cmb69) [inline reference/intl/resourcebundle.xml:52]: `ResourceBundle` implements `IteratorAggregate` only as of PHP 8.0.0 (previously, it implemented `Traversable` instead), what needs a changelog entry.

File: reference/intl/resourcebundle.xml [hunk 2]
```diff
@@ -43,20 +43,27 @@
...
+     </oointerface>
+
+     <oointerface>
+      <interfacename>Countable</interfacename>
```
Comment (cmb69) [inline reference/intl/resourcebundle.xml:56]: `Countable` is only implemented as of PHP 7.4.0, so there should be a changelog entry as well.

**Review by cmb69 — COMMENTED** [review]
Thank you!

File: reference/intl/resourcebundle.xml (hunk 1 above)
Comment (kocsismate) [reply reference/intl/resourcebundle.xml:52]: Oh, right! Added the missing change log entries

**Review by cmb69 — APPROVED** [review]
Thanks!

**Pull request merged**